prody/proteins/interactions.py

```diff
@@ -180,21 +229,69 @@ def get_energy(pair, source):
     
     lookup = pair[0]+pair[1]
     
-    return data[np.where(np.array(aa_pairs)==lookup)[0]][0][2:][np.where(np.array(sources)==source)][0]
+    try:
+        data_results = data[np.where(np.array(aa_pairs)==lookup)[0]][0][2:][np.where(np.array(sources)==source)][0]
+    except ImportError:
+        raise ImportError('Please replace non-standard names of residues with standard names.')
+
+    return data_results
+
+
+def checkNonstandardResidues(atoms):
+    """Check whether the atomic structure contain non-standard residues and inform to replace the name 
+    to the standard one to be that non-standard residues are treated in a correct way while computing 
+    interactions.
+    
+    :arg atoms: an Atomic object from which residues are selected
+    :type atoms: :class:`.Atomic`
+    """
+
+    try:
+        coords = (atoms._getCoords() if hasattr(atoms, '_getCoords') else
+                    atoms.getCoords())
+    except AttributeError:
+        try:
+            checkCoords(coords)
+        except TypeError:
+            raise TypeError('coords must be an object '
+                            'with `getCoords` method')
+    
+    amino_acids = ["ALA", "ARG", "ASN", "ASP", "CYS", "GLU", "GLN", "GLY", "HIS", "ILE", 
+                   "LEU", "LYS", "MET", "PHE", "PRO", "SER", "THR", "TRP", "TYR", "VAL"]
+
+    aa_list = atoms.select('name CA').getResnames()
+    aa_list_nr = atoms.select('name CA').getResnums()
+    nonstandard = []
+    
+    for nr_i,i in enumerate(aa_list):
+        if i not in amino_acids:
+            nonstandard.append(aa_list[nr_i] + str(aa_list_nr[nr_i]))
+    
+    LOGGER.info('There are several non-standard residues in the structure.')
+    LOGGER.info('Replace the non-standard name in the PDB file with the equivalent name from the standard one if you want to include them in the interactions.')
+    LOGGER.info("Residues: {0}.".format(' '.join(nonstandard)))   
 
 
 def showPairEnergy(data, **kwargs):
     """Return energies when a list of interactions is given. Energies will be added to each pair of residues 
     at the last position in the list. Energy is based on the residue types and not on the distances.
-    The unit of energy is kcal/mol. The energies defined as 'IB_nosolv', 'IB_solv' are taken from XX and 
-    'CS' from YY.
+    The unit of energy is kcal/mol. The energies defined as 'IB_nosolv' (non-solvent-mediated), 'IB_solv' (solvent-mediated) 
+    are taken from [OK98]_ and 'CS' from InSty paper (under preparation). 
+    Protonation of resiudues is not distinguished. The protonation of residues is not distinguished. 
```

Comment:
typo: should say `Protonation of residues` not `Protonation of resiudues` (extra u before the d)